variables.tf

```diff
@@ -110,11 +110,6 @@ variable "tags" {
   description = "Map of common tags to be placed on the resources."
   type        = map(any)
   default     = { project_name = "viya" }
-
-  validation {
-    condition     = length(var.tags) > 0
-    error_message = "ERROR: You must provide at last one tag."
-  }
```

Comment:
We really need to have strong documentation now that we are forcing a default value on the tags instead of allowing this and having folks read the documentation to let them know they need at least one tag. Just a statement

Reply:
@thpang, added a note to CONFIG-VARS.md to clarify when the tags default value will be used.

Reply:
Thomas took a look at the new CONFIG-VARS.md text and said it looks good, resolving.